src/kafka-file/languageservice/model.ts

```diff
@@ -0,0 +1,175 @@
+export interface ModelDefinition {
+    name: string;
+    description: string;
+    enum?: ModelDefinition[];
+}
+

```

Comment:
would be preferable if the documentation linked to the embedded doc instead

Reply:
I tried, but it seems that it's not supported.